node/slack-command-bot/src/utils.js

```diff
@@ -0,0 +1,44 @@
+import crypto from 'crypto';
+
+/**
+ * Throws an error if any of the keys are missing from the object
+ * @param {*} obj
+ * @param {string[]} keys
+ * @throws {Error}
+ */
+export function throwIfMissing(obj, keys) {
+  const missing = [];
+  for (let key of keys) {
+    if (!(key in obj) || !obj[key]) {
+      missing.push(key);
+    }
+  }
+  if (missing.length > 0) {
+    throw new Error(`Missing required fields: ${missing.join(', ')}`);
+  }
+}
+
+/**
+ * Throws an error if incoming request is not valid
+ * @param {*} req
+ * @throws {Error}
+ */
+export function throwIfRequestNotValid(req) {
+  const timestamp = req.headers['x-slack-request-timestamp'];
+  const bodyRaw = req.bodyRaw;
```

Comment:
I don't think this inline var is necessary.

Better to just use req.bodyRaw when needed